app/components/form/FullPageForm.tsx

```diff
@@ -87,6 +89,7 @@ export function FullPageForm<TFieldValues extends FieldValues>({
     <>
       <PageHeader>
         <PageTitle icon={icon}>{title}</PageTitle>
+        {docsPopover}
       </PageHeader>
```

Comment:
This is definitely satisfactory, but I am always looking for excuses to unbundle these giant components. And this one is easy. `title` and `icon` and `docsPopover` are only used in this header. So how about moving this whole `PageHeader` inside `app/forms/instance-create.tsx` and then we can eliminate all three props here.